### PR TITLE
Created aws-pern-voting-app example

### PR DESCRIPTION
**Pull request by jetvova**

### CODE REVIEW

File: aws-pern-voting-app/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# PERN Stack Voting App
+
+A simple voting app that uses React, Express, PostgreSQL, and NodeJS.
+
+The example shows how easy it is to deploy containers into production and to connect them to one another. Since the example defines a custom container, Pulumi does the following:
+- Builds the Docker image
```
Comment (justinvp) [inline aws-pern-voting-app/README.md:8]:
```suggestion

- Builds the Docker image
```

File: aws-pern-voting-app/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    $ pulumi config set sqlAdminName <NAME>
+    $ pulumi config set sqlsqlAdminPassword <PASSWORD> --secret
+    $ pulumi config set sqlUserName <NAME>
+    $ pulumi config set sqlUserPassword <PASSWORD> --secret
+    ```
+
+1. Restore NPM modules via `npm install` or `yarn install`.
+
+1. Run `pulumi up -y` to deploy changes:
+    ```bash
```
Comment (justinvp) [inline aws-pern-voting-app/README.md:42]:
```suggestion

    ```bash
```

File: aws-pern-voting-app/index.ts [hunk 3]
```diff
@@ -0,0 +1,175 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
```
Comment (justinvp) [inline aws-pern-voting-app/index.ts:1]:
```suggestion
// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
```

File: aws-pern-voting-app/index.ts [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+import { table } from "console";
+import { Schema } from "./PostgreSqlDynamicProvider";
+
+const config = new pulumi.Config();
+const sqlAdminName = config.require("sql-admin-name");
+const sqlAdminPassword = config.requireSecret("sql-admin-password");
+const sqlUserName = config.require("sql-user-name");
+const sqlUserPassword = config.requireSecret("sql-user-password");
+const awsConfig = new pulumi.Config("aws");
+const availabilityZone = awsConfig.get("region");
```
Comment (justinvp) [inline aws-pern-voting-app/index.ts:16]:
```suggestion
const availabilityZone = aws.config.region;
```

**Review by justinvp — APPROVED** [review]
LGTM

**Comment by cnunciato** [comment]
Merging this now to fix some broken links that are pointing to it.

**Pull request merged**